docs/zh/latest/plugins/client-control.md

```diff
@@ -23,22 +28,27 @@ title: client-control
 
 ## 描述
 
-`client-control` 插件能够动态地控制 Nginx 处理客户端的请求的行为。
+`client-control` 插件能够动态地控制 NGINX 处理客户端的请求的行为。
```

Comment:
Can you describe it more clearly? For example, by setting the size of the request body to control client access? The current description is too broad.